Userland/Libraries/LibWeb/HTML/HTMLLinkElement.cpp

```diff
@@ -115,4 +136,47 @@ void HTMLLinkElement::resource_did_load()
     document().style_sheets().add_sheet(sheet.release_nonnull());
 }
 
+void HTMLLinkElement::resource_did_load_favicon()
+{
+    VERIFY(m_relationship & (Relationship::Icon));
+    if (!resource()->has_encoded_data()) {
+        dbgln_if(SPAM_DEBUG, "Favicon downloaded, no encoded data");
+        return;
+    }
+
+    dbgln_if(SPAM_DEBUG, "Favicon downloaded, {} bytes from {}", resource()->encoded_data().size(), resource()->url());
+
+    RefPtr<Gfx::Bitmap> favicon_bitmap;
```

Comment:
This can probably be gone as well now right?

Reply:
Removed